utils/api.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+)
+
+// API Constants
+const (
+	APIUsers = "user"
+)
+
+// UserRequest is format for Request to Server
+type UserRequest struct {
+	SnykToken string `json:"snyk_api_token,omitempty"`
+	UUID      string `json:"user_id,omitempty"`
+}
+
+// UserResponse is format for Response from Server
+type UserResponse struct {
+	UUID string `json:"user_id,omitempty"`
+}
+
+// HTTPRequestType is request type to HTTPRequest Method
+type HTTPRequestType struct {
+	ReqData         UserRequest `json:"reqData,omitempty"`
+	Method          string      `json:"method,omitempty"`
+	Endpoint        string      `json:"endpoint,omitempty"`
+	ThreeScaleToken string      `json:"threeScale,omitempty"`
+	Host            string      `json:"host,omitempty"`
+	Debug           bool        `json:"debug,omitempty"`
+}
+
+// BuildAPIURL builds API Endpoint URL
+func BuildAPIURL(endpoint string, threeScale string, host string) string {
```

Comment:
- Return type must be URL, not string
- Order should be, host, endpoint, threescale
- This shouldn't exported, it never used outside this package. (i.e. start with small case)

Reply:
Ack.